packages/contracts/test/helpers/test-runner/test-runner.ts

```diff
@@ -64,24 +64,25 @@ export class ExecutionManagerTestRunner {
     StateManager: {
       owner: '$OVM_EXECUTION_MANAGER',
       accounts: {
-        ['0x4200000000000000000000000000000000000002']: {
+        [predeploys.OVM_DeployerWhitelist]: {
           codeHash: NON_NULL_BYTES32,
           ethAddress: '$OVM_DEPLOYER_WHITELIST',
         },
-        ['0x4200000000000000000000000000000000000009']: {
+        [predeploys.OVM_ProxyEOA]: {
           codeHash: NON_NULL_BYTES32,
           ethAddress: '$OVM_PROXY_EOA',
         },
       },
       contractStorage: {
-        ['0x4200000000000000000000000000000000000002']: {
-          '0x0000000000000000000000000000000000000000000000000000000000000000': getStorageXOR(
-            ethers.constants.HashZero
-          ),
+        [predeploys.OVM_DeployerWhitelist]: {
+          '0x0000000000000000000000000000000000000000000000000000000000000000': {
+            getStorageXOR: true,
+            value: ethers.constants.HashZero,
+          }
```

Comment:
This is the only section of existing code affected by this change, but the Safety Cache tests which are WIP will make heavy use of it.